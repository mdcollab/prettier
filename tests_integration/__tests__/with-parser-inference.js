"use strict";

const runPrettier = require("../runPrettier");

test("infers postcss parser", () => {
  const result = runPrettier("cli/with-parser-inference", ["*"]);

  expect(result.stdout).toMatchSnapshot();
  expect(result.status).toEqual(0);
});

test("infers postcss parser with --list-different", () => {
  const result = runPrettier("cli/with-parser-inference", [
    "--list-different",
    "*"
  ]);

  expect(result.stdout).toMatchSnapshot();
  expect(result.status).toEqual(0);
});
