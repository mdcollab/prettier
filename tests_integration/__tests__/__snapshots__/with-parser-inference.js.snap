// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`infers postcss parser 1`] = `
"/* JavaScript */
\\"use strict\\";
/* Stylesheet */
* {
  outline: none;
}
"
`;

exports[`infers postcss parser with --list-different 1`] = `""`;
